src/main/java/school/hei/haapi/model/CourseSession.java

```diff
@@ -0,0 +1,35 @@
+package school.hei.haapi.model;
+
+import static javax.persistence.GenerationType.IDENTITY;
+import java.io.Serializable;
+import java.time.Instant;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Entity
+@Table(name = "\"course_session\"")
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+public class CourseSession implements Serializable {
```

Comment:
Don't forget  `EqualsAndHashCode`  and  `ToString`
And also `Builder` might be useful

Reply:
checked